requirements-dev.txt

```diff
@@ -1,7 +1,7 @@
 black>=24.4.2
 build>=0.8.0
 cibuildwheel>=2.16.2
-clang-format>=18.1.5
+clang-format~=18.1
```

Comment:
We should add the comment we have in similar PRs, right?

Reply:
sorry, forgot to push